nuclio/triggers.py

```diff
@@ -266,6 +276,12 @@ def __init__(
             struct["attributes"]["pollingIntervalMs"] = pollingIntervalMS
         if explicit_ack_mode:
             struct["explicitAckMode"] = explicit_ack_mode
+            # workerAllocationMode conflicts with explicit_ack_mode, so we should force static one in that case
```

Comment:
Maybe raise exception / log in case of forcing it?
So user will be aware that it changed.